internal/service/service_integration/resource_service_integration_test.go

```diff
@@ -507,3 +507,67 @@ func testAccCheckAivenServiceIntegrationAttributes(n string) resource.TestCheckF
 		return nil
 	}
 }
+
+func testAccServiceIntegrationDatadogWithUserConfig(name string) string {
```

Comment:
We have the following acctest naming notation `TestAccAivenServiceIntegration_XXX` please check how other tests are done and change the name.

Reply:
It's a data provider, I have followed convention:
```
testAccServiceIntegrationResource
testAccServiceIntegrationKafkaConnectResource
testAccServiceIntegrationMirrorMakerResource
testAccServiceIntegrationLogs
testAccServiceIntegrationPreexistingReadReplica
testAccServiceIntegrationShouldFailResource
testAccServiceIntegrationDatadogWithUserConfig
```